docs/internal/admin/List-Core-Actions.md

```diff
@@ -0,0 +1,288 @@
+   - Condition: Always
+
+#### Trigger: ON_REVIEW_SUBMIT
+
+1. Adjust visibility of Review Responses (for applicant - LOQ)
+2. Change status to SUBMITTED
+3. Trim duplicated responses
+4. Update Review Statuses
+5. Increment Application Stage
+6. Generate Review Assignments
+7. Change Application Status (for applicant - LOQ)
+8. Change Application Outcome (on last stage submission)
+
+##### Actions
+
+1. Adjust visibility of Review Responses (for applicant - LOQ)
+
+   - Action: "updateReviewVisibility"
+   - Description: "**CORE-ACTION** Allow visibility to review responses (always level 1) on the current stage that were DECLINED to show to applicant after review is submitted with decision of LIST_OF_QUESTIONS"
+   - Condition: When review is_last_level & decision is LIST_OF_QUESTIONS
+
+2. Change status to SUBMITTED
+
+   - Action: "changeStatus"
+   - Description: "**CORE-ACTION** Change review status to SUBMITTED when the reviewer submits"
+   - Condition: Always
+
+3. Trim duplicated responses
+
+   - Action: "trimResponses"
+   - Description: "**CORE-ACTION** Remove duplicated and unchanged responses or ones without a decision made when review is re-submitted by the reviewer."
+   - Condition: Always
+
+4. Update Review Statuses
+
+   - Action: "updateReviewsStatuses"
+   - Description: "**CORE-ACTION** Update other review status to PENDING or CHANGES REQUESTED after one review is submitted to another reviewer in the chain of review-levels"
+   - Condition: Always
+
+5. Increment Application Stage
+
+   - Action: "incrementStage"
+   - Description: "**CORE-ACTION** Increment stage of application when last level reviewer submits review with decision of CONFORM or NON_CONFORM"
+   - Condition: When Reviewer is_last_level & decision is either CONFORM or NON_CONFORM
+
+6. Generate Review Assignments
+
+   - Action: "generateReviewAssignments"
+   - Description: "**CORE-ACTION** Generate next level (in current stage) of review assignments after one reviewer submits a review to next level - when applicable"
+   - Condition: Always
+
+7. Change Application Status (for applicant - LOQ)
+
+   - Action: "changeStatus"
+   - Description: "**CORE-ACTION** Change application status to CHANGES_REQUIRED once a last level reviewer submits with decision of LIST_OF_QUESTIONS"
+   - Condition: Reviwer is_last_level & decision is LIST_OF_QUESTIONS
+
+8. Change Application Outcome (on last stage submission)
+
+   - Action:
+   - Description: "**CORE-ACTION** Change application outcome when last level and last stage - usually a final decision - submits a review as CONFORM or NON_CONFORM. Outcome will be APPROVED or REJECTED accordingly"
+   - Condition: When Reviewer is_last_stage & is_last_level & decision is either CONFORM or NON_CONFORM
+
+9. Change Application Outcome (on Rejection in early stage)
+
+### Other actions
+
+#### Trigger: ON_APPLICATION_SUBMIT
+
+- Asynchronous console log
```

Comment:
Do we still need this? Probably not in core actions, right?

Reply:
No - see it's a different section:

## Other actions